cohortextractor/patients.py

```diff
@@ -753,6 +755,60 @@ def registered_practice_as_of(
                     },
                 },
             )
+
+
+        ### Cluster RCTs
+
+        Clustered Randomised Control Trials (Cluster RCTs) that are clustered at practice level
+        are available. There is special syntax that signifies RCT. This is `rct__trialname__propertyname`
+        for example, if there was a cluster RCT called germdefence which has a property called
+        deprivation_pctile, a variable can be created by:
+
+            germdefence_intervention=patients.registered_practice_as_of(
+                "2020-01-01",
+                returning="rct__germdefence__deprivation_pctile",
+                return_expectations={
+                    "rate": "universal",
+                    "category": {
+                        "ratios": {
+                            "1": 0.5,
+                            "2": 0.5
+                        },
+                    },
+                },
+            )
+
+        Note that a double underscore is required between `rct` and `trial_name` and between `trial_name`
+        and `property` (in this case `deprivation_pctile`).
+
```

Comment:
Not sure it's too obvious to point out: "The data resulting from the study definition will be at patient level as usual and therefore practice variables will be repeated many times for each practice, and should be aggregated in a later analysis step"